recipes/rt-utils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rt-utils" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qurit/rt-utils/archive/6c52b8d10934907ad36250d1e01894adbe5ec86f.zip
```

Comment:
Do they have a tag that reflects the version 1.2.1? Using a commit makes reduces transparency and will break the updating bot.

Reply:
@ocefpaf yes, I've switched it over to using the tag